source/features/forked-to.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import cache from '../libs/cache';
+import {getRepoURL} from '../libs/utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+const id = 'forked-to';
```

Comment:
If you need a global you might as well turn it into a function to get the full ID, I made a mistake earlier suggesting that it was only used in one place. 

Also use proper var names, I don’t know what `id` means

```suggestion
const getCacheKey = repo => 'forked-to:' + repo;
```